spinnaker-vm-simple/README.md

```diff
@@ -1,9 +1,9 @@
 # Azure Spinnaker
 
-<a href="https://aka.ms/azspindeployqs" target="_blank">
```

Comment:
Mm there was a phase where I was told "CHANGE EVERYTHING TO AKA.MS LINKS" before we knew we had telemetry. It did work, but now it's just unnecessary